### PR TITLE
Final styling tweake before beta! Line chart specifically.

### PR DESCRIPTION
**Pull request by ellenbechtel**
Changes made:
- updated data and styling for line chart
- added same-style buttons to show flow and swe to line chart
-added fill and opacity to SWEanim ridgeplot, but can take it out
-----------
Description

Testing:
----------------------------
Before making this pull request, I:
- [ ] Cleaned the code the way Vue likes it - run 'npm run lint --fix'        
- [ ] Made sure all tests run
- [ ] Ran WAVE plugin 508 compliance tool

I can confirm this has been checked on:
- [ ] Chrome
- [ ] Safari
- [ ] Edge
- [ ] Firefox
- [ ] Samsung Internet
- [ ] Internet Explorer 11 (not supported, but still needs at least a working user redirect page)

### CODE REVIEW

**Comment by ellenbechtel** [comment]
if you approve, go ahead and merge! We're basically ready!

File: public/index.html [hunk 1]
```diff
@@ -42,16 +42,36 @@
...
+            "name": "Annie Putman",
+            "email": "aputman@usgs.gov",
+            "affiliation": { "@type": "Organization", "legalName": "U.S. Geological Survey" }
+          },
+          { "@type": "Person",
+            "name": "Theodore Barnhart",
+            "email": "tbarnhart@usgs.gov",
+            "affiliation": { "@type": "Organization", "legalName": "U.S. Geological Survey" }
+          },
+          { "@type": "Person",
```
Comment (collnell) [inline public/index.html:70]: We should have Jess as the last author. I think me, you, Marty,others alphabetical, then Jess

Comment (collnell) [reply public/index.html:70]: same in IPDS

**Review by collnell — APPROVED** [review]
Can fix authors  here  later

**Pull request merged**